src/components/miscellaneous/reactions/EmojiItem.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button, Stack, Typography } from '@mui/material';
+
+import { Emoji, Event, News } from 'types';
+
+import { useCreateReaction, useDeleteReaction, useUpdateReaction } from 'hooks/EmojiReaction';
+import { useSnackbar } from 'hooks/Snackbar';
+import { useUser } from 'hooks/User';
+
+export type EmojiItemProps = {
+  data: News | Event;
+  emoji: Emoji;
+  content_type: 'news' | 'event';
```

Comment:
Finnes type for content_type. Har laget den for Comment.ts typen. Gjerne refaktorer dette til egen type og bruk den begge steder :).

Reply:
Du har ikke pushet opp PR for comment, så får ikke satt opp hvordan du har satt det opp. Vil du legge ved screen?

Reply:
Mekker det når jeg har pusha comments